diff --git a/ui/pullcounter/pullcounter.ts b/ui/pullcounter/pullcounter.ts
index 26d92f27b..1c1be633c 100644
--- a/ui/pullcounter/pullcounter.ts
+++ b/ui/pullcounter/pullcounter.ts
@@ -18,7 +18,7 @@ interface PullCounterOptions extends BaseOptions {
 
 const defaultOptions: PullCounterOptions = {
   ...UserConfig.getDefaultBaseOptions(),
-  Language: 'en',
+  Language: 'cn',
 };
 
 // NOTE: do not add more fights to this data structure.
