diff --git a/resources/user_config.ts b/resources/user_config.ts
index fbc63237..02ff276d 100644
--- a/resources/user_config.ts
+++ b/resources/user_config.ts
@@ -449,7 +449,7 @@ class UserConfig {
       de: '🔓 Entsperrt (Sperre das Overlay vor der Nutzung)',
       fr: '🔓 Débloqué (Bloquez l\'overlay avant utilisation)',
       ja: '🔓 ロック解除 (オーバーレイを使用する前にロックしてください)',
-      cn: '🔓 已解除锁定 (你需要将此悬浮窗锁定后方可使用)',
+      cn: '🔓 未锁定（使用前请在插件中勾选锁定悬浮窗）',
       ko: '🔓 위치 잠금 해제됨 (사용하기 전에 위치 잠금을 설정하세요)',
     };


diff --git a/ui/raidboss/popup-text.ts b/ui/raidboss/popup-text.ts
index 548001d3..f83dc448 100644
--- a/ui/raidboss/popup-text.ts
+++ b/ui/raidboss/popup-text.ts
@@ -94,12 +94,12 @@ const raidbossInstructions: { [lang in Lang]: string[] } = {
     'サマーフォード庄に/countdownコマンドを実行し、raidbossをテストできる。',
   ],
   cn: [
-    '请按以下步骤操作：',
-    '这是供用户调整悬浮窗大小的调试用文本',
-    '当你锁定此蓝色背景的悬浮窗',
-    '该文本即会消失。',
-    '在支持的区域中会自动加载时间轴和触发器。',
-    '可在盛夏农庄使用/countdown命令测试该raidboss模块。',
+    '使用说明',
+    '请先将本窗口拉大到合适的位置',
+    '这里会显示触发器文本',
+    '调整好后请到插件中勾选锁定悬浮窗和鼠标穿透',
+    '支持的副本会自动显示时间轴和触发器',
+    '在盛夏农庄对木人行礼会显示测试轴',
   ],
   ko: [
     '<조작 설명>',
diff --git a/ui/raidboss/html_timeline_ui.ts b/ui/raidboss/html_timeline_ui.ts
index 429fd639..50ce0244 100644
--- a/ui/raidboss/html_timeline_ui.ts
+++ b/ui/raidboss/html_timeline_ui.ts
@@ -43,12 +43,12 @@ const timelineInstructions = {
     'ラインを動的にロードする。',
   ],
   cn: [
-    '显示在此处的是',
-    '调试用时间轴。',
-    '将此悬浮窗锁定',
-    '则会立刻消失',
-    '真实的时间轴会根据',
-    '当前区域动态加载并显示',
+    '我是测试时间轴',
+    '时间轴会显示在这里',
+    '锁定之后我会消失',
+    '进入支持的副本',
+    '开场后会自动出现',
+    '无需其他设置',
   ],
   ko: [
     '이 막대바는 디버그용',
